doc/user/content/ingest-data/sql-server/self-hosted.md

```diff
@@ -11,7 +11,7 @@ aliases:
 ---
 
 This page shows you how to stream data from a self-hosted SQL Server database
-to Materialize using the [SQL Server source](/sql/create-source/sql-server/).
+to Materialize using the [SQL Server Source](/sql/create-source/sql-server/).
```

Comment:
Oh ... just realized in https://github.com/MaterializeInc/materialize/blob/main/doc/user/layouts/shortcodes/sql-server-direct/ingesting-data/allow-materialize-ips.html#L21 

We probably don't want to specify the specific database, yes? to allow for people to ingesting data from multiple database?  We probably don't want to give the impression that you need a connection per database.

Reply:
Oh actually ... I guess the `DATABASE` option is required.